common/services/allocation.js

```diff
@@ -140,34 +99,68 @@ class AllocationService extends BaseService {
     status,
     sortBy,
     sortDirection,
+    page,
   } = {}) {
     const [moveDateFrom, moveDateTo] = moveDate
 
-    return this.getAll({
+    return this.get({
       isAggregation,
       include: ['from_location', 'to_location'],
+      page,
       // TODO: This can be removed once move count and progress are returned
       // by the API as resouce meta
       includeCancelled: status ? status.includes('cancelled') : false,
-      filter: pickBy({
-        'filter[status]': status,
-        'filter[from_locations]': fromLocations.join(','),
-        'filter[to_locations]': toLocations.join(','),
-        'filter[locations]': locations.join(','),
-        'filter[date_from]': moveDateFrom,
-        'filter[date_to]': moveDateTo,
-        'sort[by]': sortBy,
-        'sort[direction]': sortDirection,
-      }),
+      filter: {
+        status: status,
+        from_locations: fromLocations.join(','),
+        to_locations: toLocations.join(','),
+        locations: locations.join(','),
+        date_from: moveDateFrom,
+        date_to: moveDateTo,
+      },
+      sort: {
+        by: sortBy,
+        direction: sortDirection,
+      },
     })
   }
 
-  getAll(props) {
-    return batchRequest(getAll, { ...props, apiClient: this.apiClient }, [
-      'from_locations',
-      'to_locations',
-      'locations',
-    ])
+  get({
+    filter = {},
+    sort = {},
+    page = 1,
+    perPage = 20,
+    includeCancelled = false,
+    isAggregation = false,
+    include,
+  } = {}) {
+    return this.apiClient
+      .all('allocation')
+      .all('filtered')
+      .post(
+        {
+          filter: this.removeInvalid(filter),
+        },
+        this.removeInvalid({
```

Comment:
Is it a git-diff formatting issue, or is `removeInvalid` a new function?
... 
*looks 1 file down*

Ah, yes, it's new :)

Reply:
Yep, it's new. And added some tests :)